src/main/target/UAVPNG030MINI/target.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+#include "drivers/io.h"
+
+#include "drivers/dma.h"
+#include "drivers/timer.h"
+#include "drivers/timer_def.h"
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    DEF_TIM(TIM3,  CH1, PA2,  TIM_USE_PPM,   0, 0), // PPM_IN_A
+    DEF_TIM(TIM3,  CH1, PA3,  TIM_USE_PPM,   0, 0), // PPM_IN_B
```

Comment:
How are the two PPM inputs meant to be used? I am pretty sure Betaflight will only listen to the first PPM input.

Reply:
Now, we really have two of them. NGOS - the original firmware for this board (https://ng.uavp.ch/Documentation/TheNGOS) - has an open concept on the number of receivers you could connect. It is able to use a serial receiver on any of the uarts as well as on two PPM-in pads on the board. See the upper right of this board picture: https://ng.uavp.ch/Documentation/Construction/HW-0.30-mini/Soldering/FlightControl?action=AttachFile&do=view&target=board-top.jpg

Reply:
In NGOS we did some drone firmware side failover stuff with multiple receivers. Eg. you could mix the two receivers data in different ways like master/slave, teacher/student and similar.

@mikeller I can change the code as proposed if it's the consens that it makes more sense that way!